src/main/java/org/opentripplanner/routing/vehicle_parking/VehicleParking.java

```diff
@@ -0,0 +1,170 @@
+package org.opentripplanner.routing.vehicle_parking;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Locale;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import org.opentripplanner.model.FeedScopedId;
+import org.opentripplanner.util.I18NString;
+
+/**
+ * Vehicle parking locations, which may allow bicycle and/or car parking.
+ *
+ * All fields are immutable except for the availability, capacity which may be updated by updaters.
+ * If any other properties change a new VehicleParking instance should be created.
+ */
+@Builder(buildMethodName = "buildInternal")
+@Getter
+@EqualsAndHashCode
```

Comment:
I think we want to remove the lombok framework from OTP. We introduced it in the `RoutingService`, but I think that was a mistake. So, we should not use it in new places.

Reply:
I've removed all usages of Lombok save for the `RoutingService`.